pkg/util/helm/helm.go

```diff
@@ -38,7 +39,8 @@ type persistenceValues struct {
 }
 
 type etcdValues struct {
-	Persistence persistenceValues `yaml:"persistence,omitempty"`
+	Persistence      persistenceValues `yaml:"persistence,omitempty"`
+	StorageContainer string            `yaml:"storageContainer,omitempty"`
```

Comment:
I think we are missing the 5 openstack service user fields here we need for the etcd secret ( see above).